presto-main/src/main/java/com/facebook/presto/cost/SymbolStatsEstimate.java

```diff
@@ -27,14 +27,7 @@
 public class SymbolStatsEstimate
 {
     private static final SymbolStatsEstimate UNKNOWN = new SymbolStatsEstimate(NEGATIVE_INFINITY, POSITIVE_INFINITY, NaN, NaN, NaN);
-
-    public static final SymbolStatsEstimate ZERO_STATS = SymbolStatsEstimate.builder()
-            .setLowValue(NaN)
-            .setHighValue(NaN)
-            .setDistinctValuesCount(0)
-            .setNullsFraction(1)
-            .setAverageRowSize(0)
-            .build();
+    private static final SymbolStatsEstimate ZERO = new SymbolStatsEstimate(NaN, NaN, 0, 0, 0);
```

Comment:
it looks like null fraction is no longer 1

Reply:
In other places through the code we use `0` as the nulls fraction when statistics are `ZERO` (for example for empty partition)

Reply:
@kokosing Aggreed. Both estimates are bad. However `0` is what we use everywhere else in the code. I feels like it has to be a separate commit changing it everywhere.

Reply:
@sopel39 I tried to check for the other places. There not that many places where we create symbol stats that are semantically equivalent to the zero stats. I thought about replacing them as well, but it seems it may decrease readability.

Actually what i think, that `zero` symbol estimate is ambiguous. It is unclear what does it mean. It somehow implies that the columns is empty. But what does `empty` means? Does it mean that the table is empty? Or does it mean that the column is all null?

I'm going to remove this constant altogether.